src/controller.cpp

```diff
@@ -526,6 +526,12 @@ void Controller::activateInternal(
   activateNext();
 }
 
+void Controller::startTimerIfInactive() {
+  if (m_connectedTimeInUTC == QDateTime()) {
```

Comment:
It would be a little more clear here to do `m_connectedTimeInUTC.isNull()` or `!m_connectedTimeInUTC.isValid()` here, because it's not exactly clear what equality to the default constructor means.

Reply:
Good call - updated this to be clearer. Thanks.